tool/test_all.sh

```diff
@@ -21,6 +22,7 @@ function run_test() {
 function run_test_flutter() {
     echo "------------- Running flutter test: $1 -------------"
     pushd $1 > /dev/null
+    rm -rf .dart_tool
```

Comment:
Are those necessary? Doing `pub upgrade` and a rebuild should have the same effects as clearing that folder, right? Does this fix anything that I'm overlooking?

Reply:
Maybe not. It is just my "just for sure" habbit. I can remove this line if you think so

Reply:
Let's remove them, that way rebuilds may be a bit faster. If it turns out to be a problem we can bring them back later.

Reply:
Okay. I removed